### PR TITLE
Add repo to a group on creation

### PR DESCRIPTION
**Pull request by jplourenco1**
Well, I am not sure that is the solution but I would like some feedback. Still trying to understand the tests.

### CODE REVIEW

File: gita/__main__.py
Comment (nosarthur) [inline gita/__main__.py:410]: maybe omit this line, so that the user has to specify a group

File: gita/__main__.py
Comment (nosarthur) [inline gita/__main__.py:85]: This check is unnecessary, line 411 already guarantees it.

File: gita/__main__.py
Comment (nosarthur) [inline gita/__main__.py:89]: also show the number of new repos added: `Added x new repo(s) to ...`

**Review by nosarthur — CHANGES_REQUESTED** [review]
lgtm, only minor changes needed

File: gita/__main__.py
Comment (jplourenco1) [reply gita/__main__.py]: right, fixed

File: gita/__main__.py
Comment (jplourenco1) [reply gita/__main__.py]: agreed

File: gita/__main__.py
Comment (jplourenco1) [reply gita/__main__.py]: done

File: gita/__main__.py
Comment (nosarthur) [inline gita/__main__.py:408]: It should not be in the exclusion group.

**Comment by jplourenco1** [comment]
about the tests, can you give me some pointers?

File: gita/__main__.py
Comment (jplourenco1) [reply gita/__main__.py]: fixed

**Comment by nosarthur** [comment]
You can skip the test for now. Otherwise you can check out test_main.py::TestAdd

You can also point the group file to `GROUP_FNAME` in the test so that some fake groups are present. As for the new repos, you can either add the gita repo, or some fake repos by mocking the APIs.

**Review by nosarthur — APPROVED** [review]
lgtm

**Comment by jplourenco1** [comment]
so i think i would like to do them in another pr, if possible, ill need to learn a bit more to do them...

**Comment by nosarthur** [comment]
sounds good, feel free to squash and merge, thanks for the contribution

**Comment by nosarthur** [comment]
fix #198

**Comment by jplourenco1** [comment]
> sounds good, feel free to squash and merge, thanks for the contribution

they are already on one commit, it is not up to you to merge?

**Comment by jplourenco1** [comment]
this is literally my first pr so sorry if should know howto do that

**Comment by nosarthur** [comment]
If you scroll down this page, there is a message from github which says "Changes approved", there is a green button with an arrow on the right.  You should be able to click that button, which says merge or something like that.

If you have many commits in the PR, the arrow next to the button can have a choice called 'squash and merge', which will squash them into 1 commit. 

Usually the contributor click that button after getting approval. I can click it if it doesn't work on your side

**Comment by jplourenco1** [comment]
i dont see this button you said, just a message "Only those with write access to this repository can merge pull requests."

**Comment by nosarthur** [comment]
I see, thanks. In now

File: gita/__main__.py
Comment (nosarthur) [inline gita/__main__.py:86]: I just realized that `repos` should be `new_repos`. I will fix them in master

**Comment by jplourenco1** [comment]
oh, ill pay more attention next time

**Pull request merged**
